warehouse/static/js/warehouse/controllers/package_manager_instructions_controller.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static values = {
+    packageName: String,
+    projectVersion: String,
+    indexUrl: String,
+    packageValue: String,
+  };
+
+  static targets = ["settings", "command"];
+
+  connect() {
+    console.log(this.packageNameValue);
+    console.log(this.projectVersionValue);
+    console.log(this.indexUrlValue);
+  }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏gonna guess these `console.log` and others go away when ready?
Will the `connect()` go away completely as well?

Reply:
yup, I left them so I don't forget to handle the project version and the index url (if this is needed)